dist/entrypoint.sh

```diff
@@ -7,6 +7,26 @@
 ACTIVATE_LICENSE_PATH="$GITHUB_WORKSPACE/_activate-license"
 mkdir -p "$ACTIVATE_LICENSE_PATH"
 
+#
+# Check if apt-get is available if in package mode (if not, we must exit since we need to install jq)
+#
+if [ "$PACKAGE_MODE" = "true" ]; then
+  echo "Checking if apt-get is installed to install jq."
+  apt-get --version > /dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    echo "apt-get is not installed. Exiting..."
+    exit 1
+  fi
+
+  # install jq
+  apt-get update \
+    &&  apt-get upgrade -y --force-yes \
+    &&  apt-get install -y --force-yes \
+        jq \
+    &&  apt-get clean \
+    &&  rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
+fi
+
```

Comment:
While this isn't wrong per se, installing JQ during CI runtime is probably not a good idea. 

I would suggest that we install JQ on the images instead, so that we can simply use it.

Also note that the `.sh` files should be as thin as possible, as the logic is all contained in the javascript part of the image (where possible).

Reply:
Makes sense! 

I'm not super well versed in Docker yet, so just to clarify - are you suggesting I modify the `Dockerfile` to install `jq` there, or modify the [`unityci/editor` image](https://hub.docker.com/r/unityci/editor) to add a `jq` installation? 

I believe you're talking about modifying the image itself - in that case, is [this repo](https://github.com/game-ci/docker) the place to do so?

Thanks for all the feedback, I really appreciate it! 😄

Reply:
Ah, only just now saw that your feedback made it so we don't need to install `jq` - my bad! I'll get to that right away! 😄